javascript/organizing_your_javascript_code/es6_modules.md

```diff
@@ -170,3 +167,9 @@ This section contains questions for you to check your understanding of this less
 - [Describe how to write an npm automation script.](https://peterxjang.com/blog/modern-javascript-explained-for-dinosaurs.html)
 - [Explain one of the main benefits of writing code in modules.](#module-knowledge-check)
 - [Explain "named exports" and "default exports".](#exports-knowledge-check)
+
+### Additional resources
+
+This section contains helpful links to other content. It isn’t required, so consider it supplemental.
+
+- [A video about ESS6 Modules](https://youtu.be/cRHQNNcYf6s) if you find video lessons easier to absorb.
```

Comment:
```suggestion
- [Watch this video about ESS6 Modules by Web Dev Simplified](https://youtu.be/cRHQNNcYf6s) if you find video lessons easier to absorb. It covers the same topics as discussed in this lesson.
```

I credited the channel in the link and thought it important to note that no extra topics were covered. If you feel there are extra topics covered, please let me know. I am also open to your feedback here of course.